advanced/Scripts/query.sh

```diff
@@ -176,11 +176,12 @@ fi
 # 2. Remove comments after domain
 # 3. Remove hosts format IP address
 # 4. Remove any lines that no longer contain the queried domain name (in case the matched domain name was in a comment)
+esc_domain="${domainQuery//./\\.}"
```

Comment:
I lifted the domain escape code from [here](https://github.com/pi-hole/pi-hole/blob/95b2560a08ec11e873e0d2190d3d0f95c7505be9/advanced/Scripts/query.sh#L28).

I considered changing `scanList()` to take an escaped domain name as an argument, but that change is not directly compatible with the development branch and IMO a bit overkill. I don't think a little code duplication will hurt.